src/inabox/amp-inabox.js

```diff
@@ -66,18 +64,17 @@ try {
 chunk(self.document, function initial() {
   /** @const {!../service/ampdoc-impl.AmpDoc} */
   const ampdoc = ampdocService.getAmpDoc(self.document);
-  /** @const {!../service/performance-impl.Performance} */
-  const perf = installPerformanceService(self);
-  perf.tick('is');
```

Comment:
This is CSI integration and I definitely don't think we should have it for in-a-box. Ultimately, there's no receptor for it in this scenario.